internal/metrics/metrics.go

```diff
@@ -69,97 +69,95 @@ func SetDmtInfo() {
 
 func SetLinterWarningsMetrics(cfg global.Global) {
 	if cfg.Linters.Templates.IsWarn() {
-		metrics.CounterAdd("dmt_linter_warnings", 1, prometheus.Labels{
-			"version":    flags.Version,
-			"linter":     "templates",
-			"id":         metrics.id,
-			"repository": metrics.repository,
+		metrics.CounterAdd("dmt_linter_info", 1, prometheus.Labels{
```

Comment:
Couldn't we use reflection here, to not miss a new linters. Something like
```
	v := reflect.ValueOf(&cfg.Linters).Elem()
        fType := v.Type()
	for i := 0; i < v.NumField(); i++ {
		field := v.Field(i)

		// Проверяем, можем ли мы создать интерфейс из этого поля
		if field.CanInterface() {
			linterConfig, ok := field.Interface().(LinterConfig)
			if ok && linterConfig.IsWarn() {
				metrics.CounterAdd("dmt_linter_info", 1, prometheus.Labels{
				"id":     metrics.id,
			        "linter": strings.ToLower(fType.Field(i).Name),
			        "level":  linterConfig.Impact.String(),
		          })
			}
		}
	}
```

@juev  @ldmonster

Reply:
Yes, we can of course. But you need to understand that this reduces productivity. Maybe it's easier to abandon the rigid structure in the configuration?

Reply:
or even without `linterConfig.IsWarn()`, just metric all levels

Reply:
DONE